core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -4749,7 +4749,7 @@ public DependencyGraph getDependencyGraph() {
             if (link.getIconFileName() == null) {
                 continue;
             }
-            if (!Jenkins.get().hasPermission(link.getRequiredPermission())) {
+            if (!Jenkins.get().hasAnyPermission(link.getRequiredPermissions().toArray(new Permission[0]))) {
```

Comment:
Don't really get this part. Is a comment needed to better understand the `new Permission[0]` or should I just RTFM?

Reply:
It’s converting a Set to an array and preserving the type, weird java quirk, I thought about adding an API that took a Set but it didn’t seem worth it for this one call